src/spyglass/utils/dj_mixin.py

```diff
@@ -863,6 +865,38 @@ def check_threads(self, detailed=False, all_threads=False) -> DataFrame:
 
         return df
 
+    # --------------------------- Check disc usage ------------------------------
+    def get_table_storage_usage(self):
+        """Total size of all analysis files in the table.
+        Uses the analysis_file_name field to find the file paths and sum their
+        sizes.
+
+        Returns
+        -------
+        tuple
+            (human-readable string, total size in bytes)
```

Comment:
I might set up `human_readable` as a boolean arg to remove the end-user need to keep track of the tuple ordering, and remove the need to calculate units when not needed